gps/index.html

```diff
@@ -10,27 +10,27 @@
-<p>A GPS network is a high precision Global Navigation Satellite System (GNSS) providing real time corrections and data for post processing. A GNSS network consists of permanently located GPS receivers, installed across an area, that generate real time high accuracy GPS positioning. Basically, all of the permanently installed GPS receivers send their data to a central system via the internet, that analyzes the input and adjusts and corrects the data to provide the most accurate locations usually down to centimeter level. Our networks use the Trimble PIVOT VRS Platform. The generated data is sent through a wireless connection using the Networked Transport of RTCM via Internet Protocol (NTRIP).</p>
+<p>A GPS network is a high-precision Global Navigation Satellite System (GNSS) providing real-time corrections and data for post processing. A GNSS network consists of permanently located GPS receivers, installed across an area, that generate real-time, high-accuracy GPS positioning. Basically, all of the permanently installed GPS receivers send their data to a central system via the internet. This central system analyzes the input and adjusts and corrects the data to provide the most accurate locations, usually down to centimeter level. Our networks use the Trimble PIVOT VRS Platform. The generated data is sent through a wireless connection using the Networked Transport of RTCM via Internet Protocol (NTRIP).</p>
 
 
-<h2 class="text-left"><i class="fas fa-fw fa-user-plus"></i> Who should be using a Network</h2>
-<p>Any person or company that wants or requires precision GPS information should be using a GPS Network. The most common applications including surveying, engineering, construction, GPS controlled machinery and GIS data collection.</p>
+<h2 class="text-left"><i class="fas fa-fw fa-user-plus"></i> Who Should Be Using a Network?</h2>
+<p>Any person or company that wants or requires precision GPS information should be using a GPS network. The most common applications of a GPS network include surveying, engineering, construction, GPS-controlled machinery, and GIS data collection.</p>
 
-<h2 class="text-left"><i class="fas fa-fw fa-signal"></i> Reference Network Benefits</h2>
+<h2 class="text-left"><i class="fas fa-fw fa-signal"></i> What Are the Benefits of Using a Reference Network?</h2>
 <img src="images/accuracy.png" class="outline flex flex--center" style="margin-bottom:2em;width:100%;max-width:778px">
-<p>A reference network eliminates the need for a reference (base) station, therefore reducing your costs. They minimizes Ionospheric errors for rovers in the network coverage. And improve efficiency and productivity in your surveys.</p>
+<p>A reference network eliminates the need for a reference (base) station, therefore reducing your costs. Reference networks minimize ionospheric errors for rovers in the network coverage and improve efficiency and productivity in your surveys.</p>
```

Comment:
"A reference network eliminates the need for a reference (base) station, therefore reducing your costs.": Reducing your cost of what? Can we be any more specific? (Should we be more specific?)

Reply:
The cost and time of the devices and users you need to get similar or better accuracy. For instance, base stations are like 25K and the consumer devices are like 1k. The base stations require a person to sit there and wait for a few hours to get a good gps reading. The gps reference network is a network of base stations.